client/python/modeldb/syncer.json

```diff
@@ -0,0 +1 @@
+../../syncer.json
```

Comment:
ModelDB doesn't currently support Windows, but if that's on the roadmap then we'll need to not rely on symlinks. Git on Windows has iffy support for turning POSIX symlinks into Windows ones, and this may not work reliably.